mantidimaging/gui/windows/stack_visualiser/view.py

```diff
@@ -74,6 +74,12 @@ def __init__(self, parent: 'MainWindowView', dock: QDockWidget, images: Images):
 
         self.image_view = MIImageView(self)
         self.image_view.imageItem.menu = self._context_actions
+
+        # Work around for https://github.com/mantidproject/mantidimaging/issues/565
+        self.image_view.scene.contextMenu = [
+            item for item in self.image_view.scene.contextMenu if item.text() != "Export..."
```

Comment:
Would it be possible to remove if it contains export i.e. item.text().lower().contains("export") opposed to exactly export because pyqtgraph might change under us and we want to make sure that the export doesn't pop back up randomly.

Reply:
Yes, that's probably safer. Fixed